trackers/javascript-tracker/src/features.ts

```diff
@@ -197,5 +147,15 @@ export function Plugins(configuration: JavaScriptTrackerConfiguration) {
     activatedPlugins.push([EventSpecificationsPlugin(), apiMethods]);
   }
 
+  if (plugins.performanceNavigationTiming) {
+    const { PerformanceNavigationTimingPlugin, ...apiMethods } = PerformanceNavigationTiming;
+    activatedPlugins.push([PerformanceNavigationTimingPlugin(), apiMethods]);
+  }
+
+  if (plugins.webVitals) {
```

Comment:
I think this should also be guarded by a `configuration.contexts` flag to be disabled by default since the default behaviour is to load an external script. I think that should be expressly opt-in behaviour for the SDK.

Reply:
Good point! Have added a configuration for the web vitals plugin.